frontend/src/routes/(authenticated)/org/[org_id]/OrgMemberTable.svelte

```diff
@@ -5,17 +5,38 @@
   import { createEventDispatcher } from 'svelte';
   import FormatUserOrgRole from '$lib/components/Orgs/FormatUserOrgRole.svelte';
   import Dropdown from '$lib/components/Dropdown.svelte';
-  import type { OrgUser, User } from './+page';
+  import { _deleteOrgUser, type Org, type OrgUser, type User } from './+page';
+  import DeleteModal from '$lib/components/modals/DeleteModal.svelte';
+  import {useNotifications} from '$lib/notify';
+  import type {LexAuthUser} from '$lib/user';
+  import {OrgRole} from '$lib/gql/types';
 
+  export let org: Org;
+  export let user: LexAuthUser;
   export let shownUsers: OrgUser[];
-  export let showEmailColumn: boolean = true;
+  export let canManage: boolean;
+
+  const { notifyWarning } = useNotifications();
 
   const dispatch = createEventDispatcher<{
     openUserModal: User,
     changeMemberRole: OrgUser,
-    removeMember: User,
```

Comment:
I pulled the removeMember GQL call into this component, because our pattern so far is to show any errors in the modal rather than closing the modal and then showing an error notification. That sort of requires the GQL and modal to be in the same place. Rather than putting them both in the +page.svelte (and bloating it more) I opted to pull them both into here.